### PR TITLE
Add IdahoFalls file

### PR DESCRIPTION
**Pull request by albeanth**
This is the beginning of the IdahoFalls file 🐃 

Please provide feedback on:

- If you have been to Idaho Falls, what did you like? What did you not like? 
- If you have never been to Idaho Falls, what interests you about coming here? 
- Are you interested in learning about lodging? What do you prefer? Hotels? AirBnb's? etc....
- @harterj , @doubwoods3 what kinds of things would you like to do? 
- @harterj , what was the name of that good 🇨🇳  food place?

### CODE REVIEW

**Comment by doubwoods3** [comment]
> @harterj , @doubwoods3 what kinds of things would you like to do?

I would like to do fun things.

**Comment by harterj** [comment]
> @harterj , @doubwoods3 what kinds of things would you like to do?
> @harterj , what was the name of that good 🇨🇳 food place?

I think I want to take a bike ride. And the restaurant is the Yummi House. We should go soon!

File: albeanth-IdahoFalls.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Idaho Falls, ID
+
+### Restaurants
+
+### Local Attractions
+
+- Hiking
+- Fishing
```
Comment (harterj) [inline albeanth-IdahoFalls.md]: I need to go get my 🐟  liscense. One of these days I'll learn how to spell liscense.

**Comment by albeanth** [comment]
@harterj @doubwoods3 I added some changes in b61927f

**Comment by doubwoods3** [comment]
> I added some changes in b61927f

@albeanth 👍 ⚡ ✊

**Comment by harterj** [comment]
I think it's pretty good. :shipit:

**Comment by doubwoods3** [comment]
> think it's pretty good.

👍

**Pull request merged**